typedb-src/modules/ROOT/pages/quickstart-guide.adoc

```diff
@@ -0,0 +1,534 @@
+----
+
+.Cluster connection example (TLS encryption enabled)
+[,bash]
+----
+typedb console --cluster=i0.deployment-a4fa2444.cloud.typedb.com:1729 --username=admin --password --tls-enabled --tls-root-ca="/Users/username/typedb-cloud/typedb-cloud-root-ca.pem"
+----
+
+To exit TypeDB Console, press kbd:[Ctrl+C].
+
+[NOTE]
+=====
+The fastest way to go through all the steps of this guide is to use the
+xref:clients:ROOT:console.adoc#_non_interactive_mode[non-interactive]
+mode by issuing the commands directly in a local terminal (not in a TypeDB Console prompt).
+=====
+--
+TypeDB Studio::
++
+--
+Connect to TypeDB:
+
+. Click the btn:[Connect to TypeDB] button on the right side of the toolbar.
+. Enter TypeDB server address (e.g., `localhost:1729`) in the *Address* field and click the btn:[Connect] button.
+
+1. Select the `TypeDB Cluster` option in the *Server* field.
+2. Click the btn:[Manage Cluster Addresses] button and add all known addresses for TypeDB servers on the cluster you
+   are connecting.
+3. Fill all other fields with appropriate information and click the btn:[Connect] button.
+
+Select the folder with the <<_download_sample_data,downloaded TypeQL files>> as a
+xref:clients:ROOT:studio.adoc#_open_a_project_directory[project directory,window=_blank] by using one of the following
+buttons:
+
+* btn:[Open Project] button in the *Project* panel (upper left);
+* btn:[Open Project Directory] button with a folder icon, leftmost on the top toolbar.
+--
+====
+
+== Create a new database
+
+To create a new database, follow the steps below.
+
+[tabs]
+====
+TypeDB Console::
++
+--
+Use the following command in a local terminal:
+
+[,bash]
+----
+typedb console --command="database create try-iam"
+----
+--
+TypeDB Studio::
++
+--
+
+. Open the *Manage Databases* dialog by clicking the database icon on the left side of the toolbar.
++
+image::studio-database.png[Database Manager button,width = 75%,link=self]
+. Enter `try-iam` in the text field, and click the btn:[Create] button next to it.
+. Now select `try-iam` from the database dropdown (next to the database icon).
+--
+====
+
+[#_define_a_schema_from_a_file]
+== Define a schema from a file
+
+Before loading any data into a new database we shall define database's schema.
+
+[tabs]
+====
+TypeDB Console::
++
+--
+Use the following command in a local terminal:
+
+[,bash]
+----
+typedb console --command="transaction try-iam schema write" --command="source iam-schema.tql" --command="commit"
+----
+--
+TypeDB Studio::
++
+--
+Open the xref:typedb::attachment$iam-schema.tql[iam-schema.tql,window=_blank] file in TypeDB Studio.
+
+Execute the define query from the file:
+
+. Ensure the btn:[Session type] (`schema` / `data`) is set to the `schema` and the
+  btn:[Transaction type] (`write` / `read`) is set to the `write` in the toolbar.
+. Run the query by clicking the btn:[Run Query] button.
+. Commit the changes in transaction by clicking the btn:[Commit Transaction] button.
```

Comment:
```suggestion
. Click btn:[Commit Transaction].
```

Reply:
We have to wait for the query execution before committing. 
```suggestion
. To commit the changes click btn:[Commit Transaction].
```